### PR TITLE
NB-48: Improve installation process

### PR DESCRIPTION
**Pull request by olgashev**

### CODE REVIEW

File: src/bin/configure-app.ts [hunk 1]
```diff
@@ -13,23 +14,25 @@ export class AppConfigurator {
...
+        console.log(chalk.cyan("\n- Now you can create your brand new own App and get the new credentials. \n- Or you can choose to use our Demo App playground which is shared accross all testers."));
+        const answer = await ui.ask("> Do you want to create your own App?", true, "y");
         if (answer === 'y') {
-            const appName = await ui.ask("Give your new app a name.", "My app");
-            const email = await ui.ask("Enter your email. We'll send you an onboarding email for your first demo tenant");
-            const proceed = await ui.ask(`We'll create a new app named '${appName}' and email '${email}'. Is that okey?`, 'y');
+            const appName = await ui.ask("> Give your new App a name.\n  This name is what your users will see when NBlocks interacts with them through emails etc.\n  App name can be changed later", true, "My App");
+            const email = await ui.ask("> Enter your email. We'll send you an onboarding email for your first demo user", false);
+            const proceed = await ui.ask(`> We'll create a new app named '${appName}' and will send onboarding email on '${email}'. Is that okey?`, true, 'y');
```
Comment (orr3) [inline src/bin/configure-app.ts:22]: email on '${email}' -> email **to** '${email}'

**Review by orr3 — CHANGES_REQUESTED** [review]

**Review by orr3 — CHANGES_REQUESTED** [review]
Small change needed

File: src/bin/configure-app.ts (hunk 1 above)
Comment (olgashev) [reply src/bin/configure-app.ts]: changed

**Review by orr3 — COMMENTED** [review]
Great

**Pull request merged**